README.md

```diff
@@ -1,2 +1,168 @@
+
+The authors of the PRs were extracted by running 
+
+```
+cat 2021-04-16_merged-or-open-prs.json \
+  | jq --raw-output '.data.repository.pullRequests.nodes[].author.login' \
+  | sort \
+  | uniq
+```
+
+#### All Authors and Co-Authors of commits on trunk
+
+Saved as [all_commits_before_2021-04-16.json](data/all_commits_before_2021-04-16.json) with 743 unique logins.
+
+<details>
+<summary>GraphQL query</summary>
+  
+```
+query($endCursor: String) {
+  repository(owner: "WordPress", name: "gutenberg") {
+    object(expression:"trunk") {
+      ... on Commit {
+        history(first:100 after:$endCursor, until:"2021-04-16T00:00:00") {
+          pageInfo {
+            hasNextPage
+            endCursor
+          }
+          nodes {
+            committedDate
+            oid
+            author {
+              email
+              name
+              user {
+                login
+              }
+            }
+            authors(first:100) {
+              pageInfo {
+                hasNextPage
+              }
+              nodes {
+                email
+                name
+                user {
+                  login
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}'
+``` 
+</details>
+
+Note that although that query only retrieves the first 100 "authors" on each commit (and does not paginate on that if there are more), I confirmed that there were not more than 100 authors on any commit (the highest was actually 8):
+
+```
+$ cat all_commits_before_2021-04-16.json \
+    | jq '.data.repository.object.history.nodes[].authors.pageInfo.hasNextPage' \
+    | sort \
+    | uniq
+false
+```
+
+The list of all "authors" on all these PR included every "author" returned here, so we are fine just including everyone listed as in the "authors" object in the above response:
+
+```
+cat all_commits_before_2021-04-16.json \
+  | jq --raw-output '.data.repository.object.history.nodes[].authors.nodes[].user.login' \
+  | sort \
+  | uniq
+```
+
+#### Resulting data
+
+All contributors for which we have GitHub logins were determined by combining the users with commits on `trunk` with the users who had opened or merged any Gutenberg PR.
+
+These users were used to initialize the `gitHubUserContributors.responses` array in [`dual-license-responses.json`](data/dual-license-responses.json).
+
+### Contributors Without An Associated GitHub Account
+
+These users were exacted from the "co-authors" of any PRs on trunk (every commit "author" was coverered by the following "coauthors" query), where the email did not have an associated github account (the "user" was null).
+
+```
+cat 2021-04-16_authors-and-coauthors-on-trunk.json \
```

Comment:
Is this file meant to be added to the repo?

Reply:
Nope, that was an oversight. I'll update this to be the `all_commits_before_2021-04-16.json` file.

Reply:
No worries 👍